models/src_iterable.yml

```diff
@@ -15,6 +15,7 @@ sources:
 
     tables:
       - name: campaign_history
+        identifier:  "{{ var('iterable_campaign_history_identifier', 'campaign_history')}}"
```

Comment:
Most of these identifiers have double spaces after the colon. I don't think it's a big issue (it should work), but could look a little strange to the end customer. Could you single space them?

Reply:
Thanks. Revised!